frontend/src/component/pages/Home.scss

```diff
@@ -14,6 +15,41 @@
         margin-right: 20px;
       }
     }
+
+    .create-short-link-btn {
+      display: flex;
+      justify-content: center;
+    }
+  }
+
+  @include respond-to($phone) {
+    .title {
+      text-align: center;
+    }
+
+    .control.create-short-link {
+      flex-direction: column;
+      height: 180px;
+
+      .text-field-wrapper {
+        display: flex;
+        margin: 4px;
+
+        input {
```

Comment:
I wonder whether we can move this media query into `https://github.com/short-d/short/blob/master/frontend/src/component/form/TextField.scss`

Reply:
Also, can we reduce the placeholder font size so as to accommodate the whole string in the input area?

Reply:
Nice idea @rohithbalaji123, i have reduced the font size from 20px to 16px. It fits completely.